modules/swagger-codegen/src/main/resources/php/ApiClient.mustache

```diff
@@ -224,7 +224,19 @@ class ApiClient
 
         // Handle the response
         if ($response_info['http_code'] == 0) {
-            throw new ApiException("API call to $url timed out: ".serialize($response_info), 0, null, null);
+            $curl_error_message = curl_error($curl);
+
+            // curl_exec can sometimes fail but still return a blank message from curl_error().
+            if (!empty($curl_error_message)) {
+                $error_message = "API call to $url failed: $curl_error_message";
+            } else {
+                $error_message = "API call to $url failed, but for an unknown reason. " .
+                    "This could happen if you are disconnected from the network.";
+            }
+
+            $exception = new ApiException($error_message, 0, null, null);
+            $exception->setResponseObject($response_info);
```

Comment:
One suggestion for enhancement in the future. We may want to add a constructor to initalize the Response Object as well, e.g.
```
            $exception = new ApiException($error_message, 0, null, null, $response_info);
```
so that the above 2 lines can be done into one without declaring a temporary variable (`$exception`)